jkube-kit/doc/src/main/asciidoc/inc/enricher/volume-permission/_jkube_volume_permission.adoc

```diff
@@ -31,4 +31,20 @@ Defaults to `777`.
 
 Defaults to `false`
 | `jkube.enricher.jkube-volume-permission.useStorageClassAnnotation`
+
+| *cpuLimit*
+| Set PersistentVolume initContainer's `.resources` CPU limit
+| `jkube.enricher.jkube-volume-permission.cpuLimit`
+
+| *memoryLimit*
+| Set PersistentVolume initContainer's `.resources` memory limit
+| `jkube.enricher.jkube-volume-permission.memoryLimit`
+
+| *cpuRequest*
+| Set PersistentVolume initContainer's `.resources` CPU request
+| `jkube.enricher.jkube-volume-permission.cpuRequest`
+
+| *memoryRequest*
+| Set PersistentVolume initContainer's `.resources` memory request
+| `jkube.enricher.jkube-volume-permission.memoryRequest`
```

Comment:
I think we should highlight that these settings apply to the `initContainer`. Maybe make it bold?